include/gl_depth_sim/interfaces/noise_model.h

```diff
@@ -0,0 +1,17 @@
+#ifndef NOISE_MODEL_H
+#define NOISE_MODEL_H
+
+#include <math.h>
+#include <vector>
+#include <iostream>
+#include <iomanip>
+#include <string>
+#include <map>
+#include <random>
+
+namespace gl_depth_sim
+{
+std::vector<float> noise(std::vector<float> distance);
```

Comment:
While I know what you're trying to do here, others will not. Instead of `noise`, please consider a name that is more descriptive. Perhaps `applyKinectNoise`. I'd also like to see a brief comment (see the other headers) that describes what this does and gives a shout-out to the paper used as a guide.